e2e/tests/join_call.spec.ts

```diff
@@ -90,10 +90,16 @@ test.describe('join call', () => {
 
         await userAPage.locator('.post__header').locator('button.user-popover').last().click();
         await expect(userAPage.locator('div.user-profile-popover')).toBeVisible();
-        await userAPage.locator('div.user-profile-popover').locator('#startCallButton').click();
+
+        // Start the call with the button
+        await expect(userAPage.getByLabel('Start call')).toBeEnabled();
+        await userAPage.getByLabel('Start call').click();
+
+        // Verify that the call button is disabled while the call is started
+        await expect(userAPage.locator('#startCallButton')).toBeDisabled();
```

Comment:
Would you be able to narrow the selector to the `div.user-profile-popover` as before, just to make sure we don't hit another element that might be added later?
Otherwise, looks great!

Reply:
I improved the locator by using more specific label, please check it out